.github/workflows/acc-tests.yaml

```diff
@@ -0,0 +1,26 @@
+name: Run Acceptance Tests
+on:
+  push:
+
+permissions:
+  contents: read
+
+jobs:
+  run-acc-tests:
+    runs-on: ubuntu-latest
+    env:
+      ATLAS_PRIVATE_KEY: ${{ secrets.ATLAS_PRIVATE_KEY }}
+      ATLAS_PUBLIC_KEY: ${{ secrets.ATLAS_PUBLIC_KEY }}
+      ATLAS_PROJECT_ID: ${{ secrets.ATLAS_PROJECT_ID }}
+      ATLAS_ORGANIZATION_ID: ${{ secrets.ATLAS_ORGANIZATION_ID }}
+
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
+        with:
+          go-version-file: .go-version
+          cache: true
+      - name: Run Acceptance Tests
+        run: make testacc
```

Comment:
Since this runs the non acceptance tests as well, I suppose it'd be redundant to have a separate step for just `make test`.

Reply:
Yeah, I had the `tests.yaml` file but ended up getting rid of it since running the acc tests runs all of them.